### PR TITLE
fix: Temporary missing block ranges fix

### PR DESCRIPTION
**Pull request by Qwerty5Uiop**
Resolves https://github.com/blockscout/blockscout/issues/9919

## Motivation

There is an issue with appearance of intersecting ranges in `missing_block_ranges` table which causes failures in `MissingRangesCollector`.

## Changelog

Updated `SanitizeMissingBlockRanges` migration to run iteratively.

Note: this is a temporary fix until we find the cause of issue

### CODE REVIEW

File: apps/explorer/lib/explorer/migrator/sanitize_missing_block_ranges.ex [hunk 1]
```diff
@@ -3,32 +3,29 @@ defmodule Explorer.Migrator.SanitizeMissingBlockRanges do
...
+  def handle_info(:sanitize, state) do
     MissingBlockRange.sanitize_missing_block_ranges()
-    MigrationStatus.set_status(@migration_name, "completed")
+    schedule_sanitize()
 
-    {:stop, :normal, state}
+    {:noreply, state}
+  end
+
+  defp schedule_sanitize do
```
Comment (mpekhota) [inline apps/explorer/lib/explorer/migrator/sanitize_missing_block_ranges.ex:28]: It looks like you have a misprint here

**Comment by mpekhota** [comment]
I builded new image with a fix, but after adding it to the chart, migrations do not start. Maybe some kind of migration is needed there?

**Comment by Qwerty5Uiop** [comment]
> I builded new image with a fix, but after adding it to the chart, migrations do not start. Maybe some kind of migration is needed there?

Hi! You mean regular DB migrations? And how do you know about it? Could you please show logs of the problem?

**Comment by vbaranov** [comment]
@mpekhota were you able to resolve the issue?

**Comment by mpekhota** [comment]
No, I couldn't fix the error after your update, I just deleted the database and ran the scan again. The new scan was successful.

**Pull request closed without merging**